app/src/main/res/values/colors.xml

```diff
@@ -1,22 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-	<color name="lightBlue600">#039be5</color>
-	<color name="lightBlue800">#0277bd</color>
+    <color name="primary">#1C93CD</color> <!--LightBlue600-->
```

Comment:
Almost missed this one, `Light Blue 600` is #039be5, `Light Blue 800` is `#0277BD`